### PR TITLE
"Fix" conveyors

### PR DESCRIPTION
**Pull request by metalgearsloth**
Turns out they were already doing the movement bandaid I thought so we probably just need to special case a couple scenarios and they'll be good enough for now.

:cl:
- tweak: Conveyors now 20% less jank.

### CODE REVIEW

**Comment by metalgearsloth** [comment]
Should be okay for now.
Known issues:
- Corner cutting
- Clickmap
- I removed the physics entirely because of the InRangeUnobstructed issue. This shouldn't really affect anything for now (long-term we'll need to use physics to subscribe to collision events to do mass conveyors remotely performantly).

File: Content.Server/Physics/Controllers/ConveyorController.cs [hunk 1]
```diff
@@ -25,43 +30,90 @@ public override void Initialize()
...
+            // TODO: This won't work if someone wants a massive fuckoff conveyor so look at using StartCollide or something.
+            foreach (var (comp, xform) in EntityManager.EntityQuery<ConveyorComponent, TransformComponent>())
             {
-                Convey(_conveyor, comp, frameTime);
+                Convey(_conveyor, comp, xform, conveyed, frameTime);
             }
         }
 
-        private void Convey(ConveyorSystem system, ConveyorComponent comp, float frameTime)
+        private void Convey(ConveyorSystem system, ConveyorComponent comp, TransformComponent xform, HashSet<EntityUid> conveyed, float frameTime)
```
Comment (PaulRitter) [inline Content.Server/Physics/Controllers/ConveyorController.cs:43]: why pass the system in using a param here? just use `_conveyor` imo

**Review by PaulRitter — APPROVED** [review]
lgtm, although i have to note: i used to have an algo in there which pushes the items on the side towards the middle. i think it was p useful, but i understand if you dont wanna add it back in bc of the headache it is :P

**Comment by metalgearsloth** [comment]
> i used to have an algo in there which pushes the items on the side towards the middle.

See I tried to add it back in but it went funny somehow, I'll try giving it a shot again before merg

**Pull request merged**